nixos/tests/nohang.nix

```diff
@@ -0,0 +1,58 @@
+# The following was modified from ./earlyoom.nix
+import ./make-test-python.nix (
```

Comment:
Please switch to the new runTest way to run tests.

Reply:
Done in [06b5667](https://github.com/NixOS/nixpkgs/commit/06b5667e0d34408f2a00bf7a43d637d2b354a861)